src/code42cli/cmds/departing_employee.py

```diff
@@ -46,7 +47,14 @@ def add(state, username, cloud_alias, departure_date, notes):
 @sdk_options()
 def remove(state, username):
     """Remove a user from the departing-employee detection list."""
-    _remove_departing_employee(state.sdk, username)
+    try:
+        _remove_departing_employee(state.sdk, username)
+    except Py42NotFoundError:
```

Comment:
in `err.response.text`, it says `Could not find item for user: <id>`... Probably want to leverage that in py42 at least.